provisioning/roles/php-memcached/meta/main.yml

```diff
@@ -1,4 +1,5 @@
 ---
 dependencies:
+  - { role: php-fpm }
```

Comment:
Ok, I'll do this in another PR and remove the dependency here, as it is a problem by itself